node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/coredump/CoredumpHandler.java

```diff
@@ -196,9 +210,16 @@ Optional<ContainerPath> enqueueCoredump(NodeAgentContext context, ContainerPath
         return Optional.of(enqueuedDir);
     }
 
+    private String corePublicKeyFlagValue(NodeAgentContext context) {
+        return coreEncryptionPublicKeyIdFlag.with(FetchVector.Dimension.NODE_TYPE, context.nodeType().name()).value();
```

Comment:
Where will the key ID come from when this has rolled out?

Reply:
Discussed this offline; key id propagation etc. will change Soon™ after some longer term design decisions have been made.